packages/cli/src/tools/packageManager.js

```diff
@@ -11,6 +11,35 @@ type Options = {|
 
 let projectDir;
 
+const packageManagers = {
+  yarn: {
+    add: ['add'],
+    addDev: ['add', '-D'],
+    remove: ['remove'],
+    install: ['install'],
+  },
+  npm: {
+    add: ['install', '--save', '--save-exact'],
+    addDev: ['install', '--save-dev', '--save-exact'],
+    remove: ['uninstall', '--save'],
+    install: ['install'],
+  },
+};
+
+function configurePackageManager(
+  packageNames: Array<string>,
+  options?: Options,
+  action: string,
+) {
+  const pm = shouldUseYarn(options) ? 'yarn' : 'npm';
+  const {yarn, npm} = packageManagers;
+  const pmConfig = pm === 'npm' ? npm : yarn;
```

Comment:
The nice thing about having package managers as keys now, is that we can directly access their config once we know which one to use:
```suggestion
  const pmConfig = packageManagers[pm];
```
line 35 is unnecessary now

Reply:
Oh didn't notice it :sweat_smile: